zenodo/modules/deposit/static/templates/zenodo_deposit/actions.html

```diff
@@ -35,22 +35,7 @@
         class="btn btn-primary"
         data-toggle="modal"
         data-target="#warningModal"
-        ng-hide="recordsVM.invenioRecordsArgs.templateParams.is_published"
-        ng-disabled="depositionForm.$invalid || depositionForm.$dirty || recordsVM.invenioRecordsLoading || (filesVM.files | checkAllFilesUploaded) || filesVM.files.length == 0"
-        ng-attr-title="{{(depositionForm.$invalid || depositionForm.$dirty || recordsVM.invenioRecordsLoading) && 'You need to save your deposit first' 
-        || (filesVM.files | checkAllFilesUploaded) && 'There are pending files to be uploaded' 
-        || (filesVM.files.length == 0) && 'You need to upload at least one file' 
-        || 'Publish the record'}}">
-        <i class="fa fa-check"></i> Publish
-      </button>
```

Comment:
There were two Publish buttons, the first one when clicked displayed the popup warning and did not perform the publish part, only showed the popup, the other button was hidden and if clicked it would not show the popup, just perform the action publish so I deleted that one and the publish button now shows the popup and only in the popup there is the action publish

Reply:
Ditto here for `ng-attr-title`, it should stay.